testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/cookies/CookiesPathTest.java

```diff
@@ -113,16 +124,15 @@ public void testMultipleCookies() throws IOException {
 
         Assert.assertThat(cookieStore.getCookies(), Matchers.hasSize(3));
 
-        CloseableHttpResponse response = login(requestURI, cookieStore);
-        response.close();
+        login(requestURI, cookieStore);
 
         // old cookie has been removed
-        // now we have AUTH_SESSION_ID, KEYCLOAK_IDENTITY, KEYCLOAK_SESSION, OAuth_Token_Request_State
-        Assert.assertThat(cookieStore.getCookies(), Matchers.hasSize(4));
+        // now we have AUTH_SESSION_ID, KEYCLOAK_IDENTITY, KEYCLOAK_SESSION
+        Assert.assertThat(cookieStore.getCookies().stream().map(c -> c.getName()).collect(Collectors.toList()), 
```

Comment:
Nit: prefer method reference to a lambda whenever possible for performance reasons (smaller memory footprint)
```suggestion
        Assert.assertThat(cookieStore.getCookies().stream().map(Cookie::getName).collect(Collectors.toList()), 
```